VungleAdapter/src/main/java/com/chartboost/mediation/vungleadapter/VungleAdapter.kt

```diff
@@ -27,47 +47,6 @@ import kotlin.coroutines.resume
  */
 class VungleAdapter : PartnerAdapter {
     companion object {
-        /**
-         * Flag to optionally set to disable Vungle's banner refresh. It must be set before the Vungle
-         * SDK is initialized for it to take effect.
-         */
-        public var disableBannerRefresh = false
-            set(value) {
-                field = value
-                PartnerLogController.log(
-                    CUSTOM,
-                    "Vungle banner ad refresh is ${if (value) "disabled" else "enabled"}."
-                )
-            }
-
-        /**
-         * Flag to optionally set to enable Vungle's mute setting. It can be set at any time and
-         * will take effect on the next ad request.
-         */
-        public var mute = false
-            set(value) {
-                field = value
-                PartnerLogController.log(
-                    CUSTOM,
-                    "Vungle mute setting is ${if (value) "enabled" else "disabled"}."
-                )
-            }
-
-        /**
-         * Flag to optionally set to enable/disable immersive mode for API 19 and newer. It can be
-         * set at any time and will take effect on the next ad request. Defaults to true.
-         *
-         * https://developer.android.com/training/system-ui/immersive.html.
-         */
-        public var immersiveMode = true
-            set(value) {
-                field = value
-                PartnerLogController.log(
-                    CUSTOM,
-                    "Vungle immersive mode is ${if (value) "enabled" else "disabled"}."
-                )
-            }
```

Comment:
Did these just vanish or do they have a newer API that we can set them on?

Reply:
They appear to have vanished. 👀 I'll keep looking, in case.

Reply:
^ @cb-jpadilla 
jinx @chauduyphanvu  :P

Reply:
Yeah, their new AdConfig looks like this:

<img width="718" alt="image" src="https://github.com/ChartBoost/chartboost-mediation-android-adapter-vungle/assets/30056771/35ee940b-d9d8-4186-9cf4-0b097785c651">

Reply:
Yep, even their docs don't specify anything in regards with those removed configurations/settings. 🤷🏼‍♂️